runtime/debug/debug.go

```diff
@@ -78,15 +78,15 @@ var (
 		Name:  "blockprofilerate",
 		Usage: "Turns on block profiling with the given rate.",
 	}
-	// CPUProfileFlag to specify where to write the CPU profile.
+	// deprecated: CPUProfileFlag
 	CPUProfileFlag = &cli.StringFlag{
 		Name:  "cpuprofile",
-		Usage: "Writes CPU profile to the given file.",
+		Usage: "Do not use, deprecated",
 	}
-	// TraceFlag to specify where to write the trace execution profile.
+	// deprecated: TraceFlag
 	TraceFlag = &cli.StringFlag{
 		Name:  "trace",
-		Usage: "Writes execution trace to the given file.",
+		Usage: "Do not use, deprecated",
 	}
```

Comment:
These should be hidden too. We don't show deprecated flags.

Add Hidden: true

Reply:
added